Doc/library/tracemalloc.rst

```diff
@@ -663,13 +663,14 @@ Traceback
    The :attr:`Trace.traceback` attribute is an instance of :class:`Traceback`
    instance.
```

Comment:
Reversing the order of frames is a major backward incompatible change for this API, please mention it:

```
.. versionchanged:: 3.7
   Frames are now sorted from the oldest to the most recent, instead of most recent to oldest.
```

Moreover, since it's a backward incompatible change, it must be documented in the "Porting" section of Doc/whatsnew/3.7.rst:

https://docs.python.org/dev/whatsnew/3.7.html#changes-in-the-python-api